Library/Formula/gpa.rb

```diff
@@ -0,0 +1,21 @@
+class Gpa < Formula
+  desc "Graphical user interface for the GnuPG"
+  homepage "https://www.gnupg.org/related_software/gpa/"
+  url "ftp://ftp.gnupg.org/gcrypt/gpa/gpa-0.9.9.tar.bz2"
```

Comment:
Please use the `https` version of this URL instead, Thanks! Upstream now support full TLS downloads, which is a nice bonus.